plugins/source/crowdstrike/codegen/recipes/incidents.go

```diff
@@ -0,0 +1,16 @@
+package recipes
+
+import (
+	"github.com/crowdstrike/gofalcon/falcon/models"
+)
+
+func CrowdScore() []*Resource {
+	resources := []*Resource{
+		{
+			Service:    "incidents",
+			SubService: "crowdscore",
+			Struct:     &models.APIMsaEnvironmentScoreResponse{},
```

Comment:
instead of using the `APIMsaEnvironmentScoreResponse` struct I think we should actually be storing the contents of
`models.APIMsaEnvironmentScoreResponse.resources` which is of type DomainEnvironmentScore ...
We don't need the errors or meta from the request stored in the destination... This way we can also use the id column as a PK

Reply:
Done